starboard/shared/x11/application_x11.cc

```diff
@@ -919,7 +919,12 @@ ApplicationX11::WaitForSystemEventWithTimeout(SbTime time) {
     return pending_event;
   }
 
-  SB_DCHECK(dev_input_);
+  // In modular builds, |CreateWindow| is not always called before the event
+  // loop is running.
+  if (!dev_input_) {
```

Comment:
The code seems fine here, but i wonder if we have other related surprises lurking somewhere. Did we miss some synchronization mechanism that should be happening ? No action needed here though, but maybe we need a quick follow-up bug to investigate.

Reply:
+1 to file a ticket with repro steps.

Reply:
Opened an internal bug with repro steps.